store/modules/events.ts

```diff
@@ -1,6 +1,8 @@
 export default {
   state: {
-    communityEvents: [],
+    activeTab: 0,
```

Comment:
Notice how the store knows about details of the view (it knows there is a tab, but could be a combo box, or a radio button or whatever), coupling both responsibilities. I'm thinking we could use a more semantic way of indicating if we want to filter past or future events by creating a type `'past'|'future'` for the active collection, replacing `activeTab`.